examples/unitTestExample/README.md

```diff
@@ -0,0 +1,4 @@
+# Setup
+
+Tests can be run via the command line without fail, however, to get them to run in Android Studio you will need to perform the following steps in your run configurations in Android Studio:
+![Unit Test Config Settings](unittest-config-settings.png)
```

Comment:
Good point. This is for the robolectric stuff with Powermock. I actually had to upgrade from 1.7.buld 71 to build 79 because of some weird issue with Powermock and the byte code generation. I'll see if I can find that link again. It was obscure, no doubt.